ts.js

```diff
@@ -69,6 +69,9 @@ module.exports = [
             '@typescript-eslint/prefer-includes': 'error',
             // Allow to use underscore as a way to ignore unused args.
             '@typescript-eslint/no-unused-vars': ['error', { 'argsIgnorePattern': '^_', 'ignoreRestSiblings': true }],
+            // Use TS extension of np-empty-function, e.g. allow to use empty constructors with parameter properties.
+            "no-empty-function": "off",
+            "@typescript-eslint/no-empty-function": "error",
```

Comment:
also mixed quoting btw

```suggestion
            'no-empty-function': 'off',
            '@typescript-eslint/no-empty-function': 'error',
```